tutorial/creating_packages/define_package_information.rst

```diff
@@ -1,2 +1,353 @@
+    hello/1.0: Package 'fd7c4113dad406f7d8211b3470c16627b54ff3af' created
+    ...
+    -- Build files have been written to: /Users/user/.conan2/p/tmp/a311fcf8a63f3206/b/build/Release
+    hello/1.0: CMake command: cmake --build "/Users/user/.conan2/p/tmp/a311fcf8a63f3206/b/build/Release" -- -j16
+    hello/1.0: RUN: cmake --build "/Users/user/.conan2/p/tmp/a311fcf8a63f3206/b/build/Release" -- -j16
+    [ 25%] Building CXX object CMakeFiles/hello.dir/src/hello.cpp.o
+    [ 50%] Linking CXX static library libhello-static.a
+    [ 50%] Built target hello
+    [ 75%] Building CXX object tests/CMakeFiles/test_hello.dir/test.cpp.o
+    [100%] Linking CXX executable test_hello
+    [100%] Built target test_hello
+    hello/1.0: RUN: tests/test_hello
+    ...
+    [ 50%] Building CXX object CMakeFiles/example.dir/src/example.cpp.o
+    [100%] Linking CXX executable example
+    [100%] Built target example
+
+    -------- Testing the package: Running test() --------
+    hello/1.0 (test package): Running test()
+    hello/1.0 (test package): RUN: ./example
+    hello/1.0: Hello World Release! (with color!)
+
+As you can see both the tests for the library and the Conan *test_package* linked against
+the *libhello-static.a* library successfully.
+
+Properties model: setting information for specific generators
+-------------------------------------------------------------
+
+The :ref:`CppInfo<conan_conanfile_model_cppinfo_attributes>` object provides the
+``set_property`` method to set information specific to each generator. For example, in
+this tutorial, we use the :ref:`CMakeDeps<CMakeDeps>` generator to generate the
+information that CMake needs to build a project that requires our library. ``CMakeDeps``,
+by default, will set a target name for the library using the same name as the Conan
+package. If you have a look at that *CMakeLists.txt* from the *test_package*:
+
+.. code-block:: cmake
+    :caption: test_package *CMakeLists.txt*
+    :emphasize-lines: 7
+
+    cmake_minimum_required(VERSION 3.15)
+    project(PackageTest CXX)
+
+    find_package(hello CONFIG REQUIRED)
+
+    add_executable(example src/example.cpp)
+    target_link_libraries(example hello::hello)
+
+You can see that we are linking with the target name ``hello::hello``. Conan sets this
+target name by default, but we can change it using the *properties model*. Let's try to
+change it to the name ``hellotarget``. To do this, we have to set the property
+``cmake_target_name`` in the package_info method of our *hello/1.0* Conan package:
+
+
+.. code-block:: python
+    :caption: *conanfile.py*
+    :emphasize-lines: 10
+
+    class helloRecipe(ConanFile):
+        ...
+
+        def package_info(self):
+            if self.options.shared:
+                self.cpp_info.libs = ["hello-shared"]
+            else:
+                self.cpp_info.libs = ["hello-static"]
+
+            self.cpp_info.set_property("cmake_target_name", "hellotarget")
```

Comment:
Add a note:
- Global target names are really not recommended, better use ppkg_name::target_name
- If the target name is pkgname::pkgname, it is NOT necessary to define it, that is already the default